### ISSUE TITLE
Add support for AcqKnowledge 4.3 files

### ISSUE THREAD

**Issue by njvack** [njvack/bioread#7]
```
In [9]: f = open("acq4.3.acq", "r")

In [10]: reader = bioread.AcqReader(f)

In [11]: reader.read()
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-11-57b35394b319> in <module>()
----> 1 reader.read()

/Users/njvack/src/bioread/src/bioread/readers.pyc in read(self)
     74             samples_per_second=self.samples_per_second)
     75 
---> 76         self._read_data()
     77         df.channels = self.channels
     78         self.data_file = df

/Users/njvack/src/bioread/src/bioread/readers.pyc in _read_data(self)
    155 
    156     def _read_data(self):
--> 157         self.channels = self.__build_channels()
    158         if self.graph_header.compressed:
    159             self.__read_data_compressed(self.channels)

/Users/njvack/src/bioread/src/bioread/readers.pyc in __build_channels(self)
    142             ch = self.channel_headers[i]
    143             cdh = self.channel_dtype_headers[i]
--> 144             data = np.empty(ch.point_count, np_map[cdh.type_code])
    145             divider = ch.frequency_divider
    146             chan_samp_per_sec = float(self.samples_per_second)/divider


KeyError: 0

In [12]: reader.file_revision
Out[12]: 124
```

**Comment by njvack** [comment]
Looks like the latest revision we support is 84, which is AcqKnowledge 4.1.1

**Comment by njvack** [comment]
Closed by 87a2d5dc1449088b1b52b3076c53ea4b3321d949
